src/main/java/com/Timos/TicTacToeInputProducer/Service/TicTacToeInputService.java

```diff
@@ -0,0 +1,16 @@
+package com.Timos.TicTacToeInputProducer.Service;
+
+import com.Timos.TicTacToeInputProducer.model.InputMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TicTacToeInputService {
+
+    @Autowired
+    private TicTacToeInputKafkaProducerService kafkaService;
+
+    public void produceTicTacToeUserInputToKafkaTopic(InputMessage inputMessage){
```

Comment:
This is likely to result in calls of the form:

`ticTacToeInputKafkaProducerService.produceTicTacToeUserInputToKafkaTopic(message)`

Clear naming is important but they should also be concise. This is the ticTacToe application, so probably doesn't need restating in the service and the method. It's a `KafkaProducerService` so the method probably doesn't need to remind me I'm sending it to Kafka. And the signature tells me that I'm providing an `inputMessage`. `.produce()` would probably be sufficient.